extensions/omni_txn/migrate/2_retry.sql

```diff
@@ -1,10 +1,12 @@
-create procedure retry(stmts text, max_attempts int default 10, repeatable_read boolean default false,
-                       collect_backoff_values boolean default false)
+create procedure retry(stmts text, max_attempts int default null,, repeatable_read boolean default false,
+                       collect_backoff_values boolean default false ,timeout interval default null )
     language c as
 'MODULE_PATHNAME';
 
 comment on procedure retry is $$
-Retry serializable transaction on statements `stmts`, `max_attempt` number of times (10 by default). `collect_backoff_values` controls if the backoff values used for sleeping will be recorded for debugging/testing purposes (false in orded to increase performance)
+Retry serializable transaction on statements `stmts`, `max_attempt` number of times (10 by default). 
+`collect_backoff_values` controls if the backoff values used for sleeping will be recorded for debugging/testing purposes (false in order to increase performance).
+`timeout` specifies the maximum time in seconds to attempt the transaction.
```

Comment:
We still need to specify expected behaviors:

* What happens if both timeout and max attempts are specified?
* Do defaults apply if one of the other values is specified?